src/test/kotlin/no/nav/k9/los/aksjonspunktbehandling/K9PunsjEventHandlerTest.kt

```diff
@@ -130,10 +131,16 @@ class K9PunsjEventHandlerTest : AbstractK9LosIntegrationTest() {
         }""".trimIndent()
 
         val event = objectMapper.readValue(json, PunsjEventDto::class.java)
-
         k9PunsjEventHandler.prosesser(event)
+
+        // V1
         val oppgaveModell = oppgaveRepository.hent(UUID.fromString(event.eksternId.toString()))
         assertFalse { oppgaveModell.aktiv }
+
+        // V3
+        val oppgaveV3 = hentV3Oppgave(event)
+        // TODO: Skal status på oppgaven endres her, eller er testen ikke relevant?
```

Comment:
Godt spørsmål. Finnes det eventer fra punsj uten aksjonspunkter? Skulle tro at sendtInn hadde vært en bedre ting å teste, men det kan skyldes midlertidig tilstand i punsj som vi ikke ønsker å vise oppgave for i los.

Reply:
Det finnes eventer uten aksjonspunkter. Dette er reelle eksempler fra dev: https://github.com/navikt/k9-los-api/blob/master/src/test/resources/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9punsjtillos/uten_fnr_dnr.json og https://github.com/navikt/k9-los-api/blob/master/src/test/resources/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9punsjtillos/prosessEventEksempel2.json

Reply:
Den siste av dem er sendt inn, så da er det forståelig at det ikke er noen aksjonspunkter.
Vet du hva slags tilstand den første eventen tilsvarer i punsj? Er den satt på vent?